tools/tgstation-server/Start Server.bat

```diff
@@ -1,24 +1,24 @@
-@echo off

```

Comment:
@kevinz000, pls.
Practice good committing. Before you hit commit, make sure you check what files are showing up as changed and make sure to revert them if they aren't the ones you meant to change before actually hitting commit. It will stop these kinds of things from happening.